pkg/systemd/services/timer-dialog.jsx

```diff
@@ -233,27 +248,58 @@ const CreateTimerDialogBody = ({ owner }) => {
                                         }}
                                         aria-label={_("Repeat")}>
                                 <FormSelectOption value="no" label={_("Don't repeat")} />
+                                <FormSelectOption value="minutely" label={_("Minutely")} />
                                 <FormSelectOption value="hourly" label={_("Hourly")} />
                                 <FormSelectOption value="daily" label={_("Daily")} />
                                 <FormSelectOption value="weekly" label={_("Weekly")} />
                                 <FormSelectOption value="monthly" label={_("Monthly")} />
                                 <FormSelectOption value="yearly" label={_("Yearly")} />
+                                <FormSelectOption value="custom" label={_("Custom")} />
                             </FormSelect>
                         </FormGroup>
                         {repeat == "no" &&
                         <FormGroup label={_("Run at")}>
                             <TimePicker className="create-timer-time-picker specific-no-repeat"
                                         menuAppendTo={() => document.body} time={specificTime} is24Hour onChange={setSpecificTime} />
                         </FormGroup>}
+                        {repeat == "custom" &&
+                        <FormGroup
+                            label={_("Custom")}
+                            validated={submitted && validateCustom ? "error" : "default"}
+                            helperTextInvalid={_("Invalid OnCalendar Event")}>
+                            <Flex>
+                                <TextInput
+                                    id="custom"
+                                    className='custom-group'
+                                    value={custom}
+                                    onChange={setCustom}
+                                    validated={submitted && validateCustom ? "error" : "default"}
+                                />
+                                <Popover key="popover-unix-group-custom" id="popover-unix-group-custom" position={ PopoverPosition.right }
+                                        headerContent={<div>Custom Timer Details</div>}
+                                        bodyContent={
+                                            <div>
+                                                <p>Any systemd calendar event.</p>
+                                                <p>IE: *:0/15</p>
+                                                <a href='https://man.archlinux.org/man/systemd.time.7#CALENDAR_EVENTS' target='_blank' rel="noreferrer">More Details</a>
```

Comment:
This needs to be localized as well as it should have icon, for example see:
![Screenshot from 2022-07-27 10-24-52](https://user-images.githubusercontent.com/12330670/181199252-af38073e-7e45-4bd4-8c80-02b92ec5033b.png)

Reply:
Also please use https://www.freedesktop.org/software/systemd/man/systemd.time.html#Calendar%20Events rather then arch wiki

Reply:
added external link icon and switched syntax to match other external links in the system.  updated link address